src/main/scala/io/treeverse/examples/List.scala

```diff
@@ -13,13 +10,17 @@ object List extends App {
       .scanLeft("")((a, b: String) => a + "/" + b)
 
   override def main(args: Array[String]) {
+    if (args.length != 2) {
+      Console.err.println("Usage: ... <repo_name> <commit_id>")
+      System.exit(1)
+    }
     val spark = SparkSession.builder().master("local").appName("I can list")
       .config("spark.hadoop.lakefs.api.url", "http://localhost:8000/api/v1/")
       .config("spark.hadoop.lakefs.api.access_key", "AKIAIOSFODNN7EXAMPLE")
       .config("spark.hadoop.lakefs.api.secret_key", "walrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY")
       .getOrCreate()
     val sc = spark.sparkContext
-    val files = LakeFSContext.newRDD(sc, "example-repo", "c7be1745d24c72020001b4954fc33661652698b07a357832ff56137f68f63b0f")
+    val files = LakeFSContext.newRDD(sc, args(0), args(1))
```

Comment:
Still prefer to name these, makes it easier to copy-paste:
```suggestion
     val repo = args(0)
     val ref = args(1)
     val files = LakeFSContext.newRDD(sc, repo, refs)
```